integration-test/src/main/java/org/apache/iotdb/it/env/DataNodeWrapper.java

```diff
@@ -88,4 +88,20 @@ protected void addStartCmdParams(List<String> params) {
   protected String mainClassName() {
     return "org.apache.iotdb.db.service.DataNode";
   }
+
+  public int getMppDataExchangePort() {
```

Comment:
Have these methods been used?

Reply:
the methods added in DataWrapper will be used in DataNode test